thousandeyes-sdk-core/src/thousandeyes_sdk/core/configuration.py

```diff
@@ -79,37 +41,6 @@ def __init__(self, host=None,
         self._base_path = "https://api.thousandeyes.com" if host is None else host
         """Default Base url
         """
-        self.server_index = 0 if server_index is None and host is None else server_index
```

Comment:
Can you explain the rationale for what we decided could be removed?

Reply:
Well we didn't actually decide on anything in specific so I just looked at whatever seemed like we didn't need. Also chatted with Jack before to confirm some things and point out some more possible removals.
We talked by message so I do have his rationale instead. 
In case you wan't to see i, I can share it if @jabrowne-te doesn't mind